### PR TITLE
fix: team page member sort

### LINKED ISSUE #845: [BUG]: team page member order
## Description

In the team details page the listing of the team's members is not appearing in the correct order.

image

### Steps to reproduce

Go to the details page of any team.

### Expected result

The list of team members should be ordered alphabetically

### Actual result

The list of team members is not ordered in any way.

### PR DESCRIPTION
**Pull request by JoaoSaIvador**
Relates to #845
Fixes #845

## Screenshots

image

## Proposed Changes

  - The list of team members is ordered alphabetically

This pull request closes #845

### CODE REVIEW

File: backend/src/modules/teams/services/get.team.service.ts [hunk 1]
```diff
@@ -23,8 +23,24 @@ export default class GetTeamService implements GetTeamServiceInterface {
 		return this.teamRepository.countDocuments();
 	}
 
-	getTeam(teamId: string) {
-		return this.teamRepository.getTeam(teamId);
+	async getTeam(teamId: string) {
+		const team = await this.teamRepository.getTeam(teamId);
+		const users = await this.teamUserRepository.getUsersOfTeam(teamId);
```
Comment (rpvsilva) [inline backend/src/modules/teams/services/get.team.service.ts:28]: Here you can use a `Promise.all` to resolve the promises at the same time

File: backend/src/modules/teams/services/get.team.service.ts [hunk 2]
```diff
@@ -23,8 +23,24 @@ export default class GetTeamService implements GetTeamServiceInterface {
...
+			return fullNameA < fullNameB ? -1 : 1;
+		});
+
+		const teamWithUsers = {
+			_id: team._id,
+			name: team.name,
+			users
+		};
+
+		return teamWithUsers;
```
Comment (rpvsilva) [inline backend/src/modules/teams/services/get.team.service.ts:43]:
```suggestion
		return {
			_id: team._id,
			name: team.name,
			users
		};
```

**Review by rpvsilva — CHANGES_REQUESTED** [review]

File: backend/src/modules/teams/services/get.team.service.ts (hunk 1 above)
Comment (JoaoSaIvador) [reply backend/src/modules/teams/services/get.team.service.ts]: Nuno had requested previously that we make only one request, so we changed that

**Review by rpvsilva — APPROVED** [review]

**Review by nunocaseiro — APPROVED** [review]
LGTM 🚀

**Pull request merged**